README.rst

```diff
@@ -134,6 +134,28 @@ Your buildout should be ready to use now.
 See `the documentation of mr.bob <http://mrbob.readthedocs.org/en/latest/>`_  for further information.
 
 
+Integration with plone.recipe.codeanalysis
+------------------------------------------
+
+This package is integrated by default with ``plone.recipe.codeanalysis`` that enforces the use of good known practices and the Plone code style guide.
+
+By default, the recipe setup a git precommit hook that performs the checks on the current package code. If it fins some violation the commit is not performed and the violations are shown.
```

Comment:
I double checked and you're absolutely right. Sorry I don't know how I looked into it before it seemed to me that the commit was not being applied because of the config of the pre-commit hook... in fact, the motivation for the whole PR was for fix it.